app/models/miq_region.rb

```diff
@@ -189,49 +189,35 @@ def remote_ui_miq_server
   end
 
   def remote_ui_ipaddress
-    server = remote_ui_miq_server
-    server.try(:ipaddress)
+    remote_ui_miq_server&.ui_ipaddress
   end
 
   def remote_ui_hostname
-    server = remote_ui_miq_server
-    server && (server.hostname || server.ipaddress)
+    remote_ui_miq_server&.ui_hostname
   end
```

Comment:
NOTE these methods don't appear to be used outside of the remote_*_address/url methods but since they aren't private I don't want to remove them especially for something we might backport.

[`remote_ui_ipaddress`](https://github.com/search?q=org%3AManageIQ%20remote_ui_ipaddress&type=code), [`remote_ui_hostname`](https://github.com/search?q=org%3AManageIQ%20remote_ui_hostname&type=code), [`remote_ws_ipaddress`](https://github.com/search?q=org%3AManageIQ%20remote_ws_ipaddress&type=code), [`remote_ws_hostname`](https://github.com/search?q=org%3AManageIQ%20remote_ws_hostname&type=code)